### ISSUE TITLE
Review limit setting behaviour and allow usage in tandem with the Retina

### ISSUE THREAD

**Issue by selul** [Codeinwp/optimole-wp#767]
### What problem does this address?

Right now if the Limit setting behavior is on, the retina option can not be used, this being the default behavior. 

We added this to prevent when the dpr size is exceeding the limit in this case, however this is bringing a loss in quality for the images for customers that prefer a best image quality within the constrain of web vitals. 

I got this request from various customers that complained about their issue with image compression and after enabling the DPR they were happy with the quality. 

### What is your proposed solution?

I would suggest disconnecting those settings dependecnies and apply the limit to the size before DPR like is doing right now, however leaving the DPR factor to be applied. 

Moreover, we can also enable the DPR factor aswell by default. 

@kushh23 @abaicus curious if you have any feedback. 

### Will this feature require documentation? (Optional)

None

**Comment by kushh23** [comment]
I agree with you that it is creating confusion for the users as the Limit settings are turned on by default on the first install, and due to this, retina is disabled, hence resulting nonretina images site, which sometimes looks blurry. Many users have asked why the retina is not working, and the reason is limit size option being on. By disconnecting those two settings, we can avoid this confusion.

**Comment by abaicus** [comment]
@kushh23 no, it is not implemented in today's release.

**Comment by kushh23** [comment]
@abaicus Okay, thanks for the information. I read the message on slack about the release, so got confused.

**Comment by ineagu** [comment]
thanks for linking the issues @kushh23.  Personally I enable retina on the sites we are using and for me it can be enabled by default since there is a big difference in UX.

Also I won't use the limit image sizes myself since we can already use the WordPress setting to load an image at a lower size if this is what we want.

**Comment by selul** [comment]
This is done here https://github.com/Codeinwp/optimole-wp/pull/905

Will require a release here aswell https://github.com/Codeinwp/optimole_js_lib/pull/131

**Comment by vytisbulkevicius** [comment]
@selul, since https://github.com/Codeinwp/optimole_js_lib/pull/131 is merged and released I should be able to test this with both builds from https://github.com/Codeinwp/optimole-wp/pull/905#issuecomment-2758635466, correct? (I tried with both)

Because I tried on multiple test websites and it doesn't seem to work for me.
Short screencast: https://vertis.d.pr/v/9DWryF

Post tested: https://luxuriant-peacock-836c37.vertisite.cloud/conquering-the-marketing-maze-creative-strategies-on-a-budget/

> Site: https://luxuriant-peacock-836c37.vertisite.cloud/wp-admin
> Username: dujayobeka3804
> Password: dW6ElhF8HarkJ9N2TiBw

**Comment by selul** [comment]
@vytisbulkevicius great catch, seems like the library wasnt released. now should be good.
